src/js/angular/sparql-template/controllers.js

```diff
@@ -276,7 +277,7 @@ function SparqlTemplateCreateCtrl(
             }).catch(function (data) {
                 const message = getError(data);
                 toastr.error(message, $translate.instant('save.sparql.template.failure.msg', {templateID: templateInfo.templateID}));
-                throw new SparqlTemplateError();
+                return Promise.reject(new SparqlTemplateError());
```

Comment:
I'd suggest that we have concrete error messages for every specific case where this SparqlTemplateError is thrown

Reply:
Done